testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -95,11 +95,23 @@ pub struct EmitJobRequest {
 
 impl EmitJobRequest {
     pub fn for_instances(instances: Vec<Instance>) -> Self {
-        Self {
-            instances,
-            accounts_per_client: 10,
-            threads_per_ac: None,
-            thread_params: EmitThreadParams::default(),
+        if instances.len() == 100 {
+            Self {
+                instances,
+                accounts_per_client: 16,
+                threads_per_ac: Some(2),
+                thread_params: EmitThreadParams {
+                    wait_millis: 0,
```

Comment:
I think we can try to unify those params so that we don't have special case for 100 nodes
- wait_millis can be set to 0 for sure, this was probably bad idea to have it as non-0 by default
- For `threads_per_ac` we can just boost multiplier from `150` to `200` in function that evaluates it based on node count
- `accounts_per_client` can probably also be just set to 16 by default

Reply:
Updated